src/Servers/Kestrel/shared/test/ServerRetryHelper.cs

```diff
@@ -7,6 +7,8 @@ namespace Microsoft.AspNetCore.Testing;
 
 public static class ServerRetryHelper
 {
+    private const int RetryCount = 15;
```

Comment:
nit: There's no delay in the loop below beyond connection timeouts. Suggest going higher if I'm right that address in use normally causes an almost-immediate failure.